modules/planning/open_space/trajectory_smoother/distance_approach_ipopt_fixed_dual_interface.cc

```diff
@@ -777,7 +777,6 @@ void DistanceApproachIPOPTFixedDualInterface::eval_constraints(int n,
   g[constraint_index + 2] = x[state_index + 2];
   g[constraint_index + 3] = x[state_index + 3];
   constraint_index += 4;
-  state_index += 4;
```

Comment:
It might be better to leave it here to make it more easier to debug. In case equation matrix needs to be modified for re-formulated later.